lib/hexpm_web/templates/package/show.html.eex

```diff
@@ -161,6 +161,12 @@ tools = [
 </div>
 
   <div class="col-md-3">
+    <h3>Report</h3>
+      <form action = "<%= Routes.package_report_path(Endpoint, :new) %>">
+        <input type = "hidden" name = "package" value = "<%= @package.name %>"/>
+        <input type = "hidden" name = "repository" value = "<%= @package.repository.name %>"/>
+        <button type = "submit"> Report <%= ViewHelpers.package_name(@package) %> </button>
+      </a>
```

Comment:
Don't add spaces around `=`. Try to follow the formatting in the rest of the project.